docker-compose.yml

```diff
@@ -2,8 +2,9 @@ version: '2'
 
 services:
   jekyll:
-    image: jekyll/jekyll:4.2.0
-    command: jekyll serve --watch --force_polling --verbose --livereload
+    image: jekyll/jekyll:4.2.2
```

Comment:
Do we want to set the same version that we have in the Gemfile?

Reply:
This version is from the docker repository of a contributor, this isn't the version of jekyll, but this docker uses the version that is set on the gemfile.